.github/workflows/fly-deploy.yml

```
@@ -1,18 +1,61 @@
+        required: true
+        type: string
+        default: 'manual'
+
 jobs:
-  deploy:
-    name: Deploy app
+  deploy-honcho-image:
+    name: Deploy Honcho Image (Test Environment)
     runs-on: ubuntu-latest
-    concurrency: deploy-group    # optional: ensure only one action runs at a time
+    concurrency: 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)